docs/cloud/how-to-create-a-namespace-in-temporal-cloud.md

```diff
@@ -0,0 +1,46 @@
+---
+id: how-to-create-a-namespace-in-temporal-cloud
+title: How to create a Namespace in Temporal Cloud
+sidebar_label: Create a Namespace
+description: To create a Namespace in Temporal Cloud, use either Temporal Web UI or tcld.
+tags:
+  - how-to
+---
+
+:::info
+
+The user who creates a [Namespace](/namespaces) is automatically granted [Namespace Admin](/cloud/what-are-the-namespace-level-permissions-for-users-in-temporal-cloud) permission for that Namespace.
+
+:::
+
+<!--- What information do I need to create a Namespace in Temporal Cloud? --->
+
+## Information needed to create a Namespace
+
+To create a Namespace in Temporal Cloud, gather the following information:
+
+- [Namespace Name](/cloud/#cloud-namespace), region, and environment
+- [Retention period](/clusters/#retention-period) for [Event History](/workflows#event-history)
+- [End-entity certificate](/cloud/how-to-manage-certificates-in-temporal-cloud#requirements-for-client-certificates) for connecting to the Namespace
```

Comment:
To create a namespace, the user needs to provide the CAs, not the end-entity certificate. The end-entity certificate can be issued later when the user wants to connect to their namespace using tctl or the SDKs